docs/src/main/sphinx/connector/delta-lake.md

```diff
@@ -708,6 +708,16 @@ WITH (
 )
 AS SELECT name, comment, regionkey FROM tpch.tiny.nation;
 ```
+(delta-lake-shallow-clone)=
+#### Shallow clone
```

Comment:
Since it is just tables.. maybe we should consistently use the term "Shallow cloned tables"